lib/elixir/lib/application.ex

```diff
@@ -145,6 +145,19 @@ defmodule Application do
   """
   @callback stop(state) :: term
 
+  @doc """
+  Called after an application starts.
```

Comment:
I think the single line description would be more like "Start an application in synchronous phases".

The we describe it afterwards. Note that can use start phases without any included applications.